### PR TITLE
Update gradle versions.

### PR DESCRIPTION
**Pull request by IgorGanapolsky**
Use common ext block.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,10 +5,17 @@ buildscript {
...
-        classpath 'com.android.tools.build:gradle:1.1.0'
+        classpath 'com.android.tools.build:gradle:1.1.3'
     }
 }
 
+// To avoid manually setting the same values in all Android modules, set the value on the root
+// project and then reference this from the modules
+ext {
+    compileSdkVersion = 22
+    buildToolsVersion = "22"
```
Comment (mikepenz) [inline build.gradle]: the newest version is 22.0.1 :D thanks

**Comment by IgorGanapolsky** [comment]
@mikepenz Thanks for the nice library.  Have some @changetip $1 bitcoin on me.

**Comment by changetip** [comment]
Hi @mikepenz, @IgorGanapolsky sent you a Bitcoin tip worth 4,247 bits ($1.00), and I'm here to deliver it ➔ **[collect your tip](https://www.changetip.com/collect/735064)**.


**[Learn more about ChangeTip](https://www.changetip.com/tip-online/github)**

**Comment by mikepenz** [comment]
thanks :)

**Pull request merged**